grafana_crd.yaml

```diff
@@ -0,0 +1,5727 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Looks like cruft?

Reply:
This is part of the Grafana operator CRD, no changes were made.

Reply:
you are correct, removed.